t/utils/utils.go

```diff
@@ -336,10 +337,36 @@ func ImportTestAccount(keystoreDir, accountFile string) error {
 	}
 
 	dst := filepath.Join(keystoreDir, accountFile)
-	err := ioutil.WriteFile(dst, static.MustAsset("keys/"+accountFile), 0644)
+	err := copyFile(dst, path.Join(params.GetStatusHome(), "static/keys/", accountFile))
 	if err != nil {
 		logger.Warn("cannot copy test account PK", "error", err)
 	}
 
 	return err
 }
+
+func gettestConfigFromFile(fileName string, config *testConfig) error {
+	configData, err := ioutil.ReadFile(fileName)
+	if err != nil {
+		return err
+	}
+
+	if err := json.Unmarshal(configData, &config); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func copyFile(sourceFile string, destinationFile string) error {
```

Comment:
It's fine as files are super small but if you want to play a bit with this code, I believe it can rely exclusively on streams to copy a file instead of loading it into the memory and then dumping the memory somewhere else.